lib/customer.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'restaurant'
+
+class Customer
+
+  attr_accessor :order, :total_cost
+
+  def initialize
+    @order = []
+    @total_cost = 0
+  end
+
+  def see_all_dishes(restaurant)
+    restaurant.dishes.each do |item, price|
+      # price = sprintf('%.2f', price)
+      p "#{item} - £#{sprintf('%.2f', price)}"
```

Comment:
By using "p" you're tying the presentation of your system to the terminal. What if I wanted to have this text displayed on a web page?